### PR TITLE
Build add menu by context with test

### PR DESCRIPTION
**Pull request by oshri**
Before running, install new dependencies. ( Yarn )
I add jest __snapshots__ in createMenuComponent tests.

### CODE REVIEW

File: src/app/modules/common/components/createMenu/createMenu.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <div  class="menu-options"
+        [@showHide]="showMenu"
+        fxLayout="row"
+        fxLayoutAlign="start center">
+
+    <button class="circle-border-button"
+            *ngFor="let option of options"
+            [matTooltip]="option"
+            (click)="onSelectAction(option)">
+      {{firstLetterFromType(option)}}
```
Comment (Nexxado) [inline src/app/modules/common/components/createMenu/createMenu.component.html]: try avoiding function calls in template.
With function calls, Angular's change detection engine cannot determine if the value changed and has to run the function every change detection cycle.

maybe add it as a property for each option in the `options` array?

File: src/app/modules/common/components/rtfItem/rtfItem.component.ts [hunk 2]
```diff
@@ -92,4 +92,8 @@ export class OkRtfItemComponent implements OnInit {
         this.add.emit({index: this.index, type: this.type});
     }
 
+    public createAction(event: {type: FORM_TYPE}): void {
+      console.log(event.type);
+    }
```
Comment (Nexxado) [inline src/app/modules/common/components/rtfItem/rtfItem.component.ts]: Does this function suppose to create an action? it only `console.log`s the type.

Comment (Nexxado) [reply src/app/modules/common/components/rtfItem/rtfItem.component.ts]: You mean in a future feature/pull-request?

Comment (nightpost) [reply src/app/modules/common/components/rtfItem/rtfItem.component.ts]: @Nexxado near pull request hahaha

**Pull request merged**